webapp/public/javascripts/angular/data-series/registration.js

```diff
@@ -2198,12 +2198,13 @@ define([], function() {
 
       var getAliasFromCsvFields = function(fieldType, fields){
         var fieldAlias;
-        fields.forEach(function(field){
-          if (field.type == fieldType){
-            fieldAlias = field.alias;
-            return;
+	var field;
+	for (field in fields){
```

Comment:
Não é necessário inicializar a variavel do for desta maneira. Pode colocar diretamente como `for(var field in fields`.

Poderia também iterar sobre os elementos ao inves do index com a seguinte sintaxe:
```js
for(let field of fields) {
  
}
```